src/configure/configure.ts

```diff
@@ -478,12 +478,12 @@ class Orchestrator {
                 throw error;
             }
         }
-        else if (!!node.cloudResource && node.cloudResource.nodeType === 'cluster') {
-            this.inputs.subscriptionId = node.cloudResource.subscription.subscriptionId;
+        else if (node.resource.nodeType === "cluster") {
```

Comment:
We should use same interface which we created in AKS extension. The node object should be of that type instead of AzureTreeitem

Reply:
In fact we should use this type when we are registering the command itself in https://github.com/microsoft/vscode-deploy-azure/blob/fdf6af7893f9eadc59c79e0bd1636bc01931b9be/src/configure/activate.ts#L25

Reply:
We will need to use `IResourceNode | any` to accomodate folder right click.

Can we also model right click as an interface instead of `any`?

Reply:
The folder structure is way different from the resource structure. So at the initial stage, have kept ``` any ``` and where we are dealing the the resource scenario, have used ``` IResourceNode ```.